appservice/src/SiteClient.ts

```diff
@@ -456,13 +466,13 @@ export class SiteClient implements IAppSettingsClient {
         }));
     }
 
-    public async vfsPutItem(context: IActionContext, data: string | ArrayBuffer, url: string, rawHeaders?: {}): Promise<AzExtPipelineResponse> {
+    public async vfsPutItem(context: IActionContext, data: string | ArrayBuffer, url: string, rawHeaders?: Record<string, string>): Promise<AzExtPipelineResponse> {
         const client: ServiceClient = await createGenericClient(context, this._site.subscription);
         const headers = createHttpHeaders(rawHeaders);
         return await client.sendRequest(createPipelineRequest({
             method: 'PUT',
             url,
-            body: typeof data === 'string' ? data : data.toString(),
+            body: typeof data === 'string' ? data : data.toString(), // TODO: This isn't right
```

Comment:
TODO: Needs to be fixed

Reply:
Might need input from @nturinski on this one. If the input is an `ArrayBuffer` (or `Uint8Array`), then the `.toString()` output will be something like `'10,20,30,255'`, for `[10, 20, 30, 255]`. I doubt this is what the service is expecting, but it has been like this for a couple years.